web-registry/src/components/organisms/EcocreditsTable.tsx

```diff
@@ -0,0 +1,217 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
most of the code here is copied over from the prior `Ecocredits.tsx` - minus the render prop to display buttons